.travis/scripts.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
```

Comment:
How is that different? This script also will be run only from travis, I don't see a problem...or am I missing something?